test/unit/range/views/istreambuf_test.cpp

```diff
@@ -46,39 +45,32 @@ INSTANTIATE_TYPED_TEST_SUITE_P(iterator_fixture, iterator_fixture, test_type, );
 
 TEST(view_istreambuf, basic)
 {
+    using namespace std::literals;
```

Comment:
Is this for using this `operator""sv` thing?

Reply:
Yes, we also want to do `seqan3::literals`. See https://github.com/seqan/product_backlog/issues/287#issuecomment-775223706 for some reasoning

Reply:
Ah so we can use it for all operator""... stuff. Nice!